agent/consul/acl.go

```diff
@@ -212,6 +213,8 @@ type ACLResolverConfig struct {
 	Tokens *token.Store
 }
 
+const aclClientDisabledTTL = 30 * time.Second
```

Comment:
Am I reading this right that the old "default" in `defaults.go` which was actually never overridable was 120 seconds but now we are setting it to be always 30? Does that actually change the behaviour?

If it does, perhaps some commentary in the PR about why the new value is better/safe etc. is worthwhile and a changelog entry would be good? But maybe I'm just misunderstanding the PR as I don't recall this code at all!

Reply:
> Am I reading this right that the old "default" in defaults.go which was actually never overridable was 120 seconds but now we are setting it to be always 30? Does that actually change the behaviour?

No, the 120s was always ignored. See https://github.com/hashicorp/consul/pull/10808#discussion_r684543725 (I've been trying to add that to the PR description but github is having issues).

It was using the 30s set in `NewConsulConfig`, so this preserves the old behaviour.